README.md

```diff
@@ -1,826 +1,73 @@
-| `requestOptions` | `RequestOptions` | Optional | Pass additional request options. |
-
-##### Response Type
-
-`void`
-
-##### Example Usage
-
-```ts
-const accountId = 'accountId0';
-const sessionId = 'sessionId8';
-const participantId = 'participantId0';
-const body: Subscriptions = {
-  sessionId: 'd8886aad-b956-4e1b-b2f4-d7c9f8162772',
-};
-
-try {
-  const { result, ...httpResponse } = await apiController.addParticipantToSession(accountId, sessionId, participantId, body);
-  // Get more response info...
-  // const { statusCode, headers } = httpResponse;
-} catch(error) {
-  if (error instanceof ApiError) {
-    const errors = error.result;
-    // const { statusCode, headers } = error;
-  }
-}
+const client = new Client({
+  basicAuthUserName: 'user',
+  basicAuthPassword: 'pass'
+})
 ```
 
-##### Errors
-
-| HTTP Status Code | Error Description | Exception Class |
-|  --- | --- | --- |
-| 401 | Unauthorized | `ApiError` |
-| 403 | Access Denied | `ApiError` |
-| 404 | Not Found | `ApiError` |
-| Default | Unexpected Error | [`Error`](#error) |
-
-#### Remove Participant From Session
-
-Remove a participant from a session
-
-This will automatically remove any subscriptions the participant has associated with this session
+## Create Session
 
-```ts
-async removeParticipantFromSession(
-  accountId: string,
-  participantId: string,
-  sessionId: string,
-  requestOptions?: RequestOptions
-): Promise<ApiResponse<void>>
 ```
+import { Client, Session, SessionsController } from '@bandwidth/webrtc';
 
-##### Parameters
-
-| Parameter | Type | Tags | Description |
-|  --- | --- | --- | --- |
-| `accountId` | `string` | Template, Required | Account ID |
-| `participantId` | `string` | Template, Required | Participant ID |
-| `sessionId` | `string` | Template, Required | Session ID |
-| `requestOptions` | `RequestOptions` | Optional | Pass additional request options. |
-
-##### Response Type
-
-`void`
-
-##### Example Usage
-
-```ts
+const client = new Client({
+  basicAuthUserName: 'user',
+  basicAuthPassword: 'pass',
+})
+const sessionsController = new SessionsController(client);
 const accountId = 'accountId0';
-const participantId = 'participantId0';
-const sessionId = 'sessionId8';
-try {
-  const { result, ...httpResponse } = await apiController.removeParticipantFromSession(accountId, participantId, sessionId);
-  // Get more response info...
-  // const { statusCode, headers } = httpResponse;
-} catch(error) {
-  if (error instanceof ApiError) {
-    const errors = error.result;
-    // const { statusCode, headers } = error;
-  }
-}
-```
-
-##### Errors
-
-| HTTP Status Code | Error Description | Exception Class |
-|  --- | --- | --- |
-| 401 | Unauthorized | `ApiError` |
-| 403 | Access Denied | `ApiError` |
-| 404 | Not Found | `ApiError` |
-| Default | Unexpected Error | [`Error`](#error) |
-
-#### Get Participant Subscriptions
+const body: Session = {};
+body.id = '75c21163-e110-41bc-bd76-1bb428ec85d5';
```

Comment:
I copy-pasted most of this from the autogenerated docs from our OpenAPI spec. The same object is used for the request and response in the spec, so that's probably kinda not true. We'll probably need to split those up in order to accurately show what can be used for a request, and what can be used for a response.

As for the readme update, I'll just remove these